include/cpp/span.hpp

```diff
@@ -114,9 +114,9 @@ inline void contract_violation(const char* msg)
 #define TCB_SPAN_HAVE_DEDUCTION_GUIDES
 #endif
 
-#if defined(TCB_SPAN_HAVE_CPP17) || defined(__cpp_lib_byte)
-#define TCB_SPAN_HAVE_STD_BYTE
-#endif
+//#if defined(TCB_SPAN_HAVE_CPP17) || defined(__cpp_lib_byte)
+//#define TCB_SPAN_HAVE_STD_BYTE
+//#endif
```

Comment:
Can you explain why these lines have been commented out?

Reply:
This was due to a name collision between tcb::byte and a byte typedef in MS MIDL definitions dragged in by using <Unknwn.h>

The root cause of this is "using namespace tcb" in <audio>, which can be fixed

Reply:
Fixed in 7ec6cc8